lib/pure/collections/deques.nim

```diff
@@ -84,6 +84,12 @@ proc initDeque*[T](initialSize: int = 4): Deque[T] =
   ## The length of a newly created deque will still be 0.
   result.initImpl(initialSize)
 
+proc initDeque*[T](s: seq[T]): Deque[T] =
+  ## Create a new deque initialized with a sequence
+  result = initDeque[T]()
+  for i in 0..< s.len:
```

Comment:
Nitpick: use consistent spacing around ..< - either space before/after it, or no spaces at all